elastic/kibana
# Stack monitoring - update standalone cluster query for metricbeat 8.0

### Summary
Standalone cluster is not showing up with metricbeat 8 data because we target metricsets that don't exist ('logstash_stats', 'logstash_state', 'beats_stats', 'beats_state'). The `metricset.name` we filter on only contains `stats`, `state`, `node_stats` or `node`. Thankfully each metricset has an [event.dataset](https://github.com/elastic/beats/blob/master/metricbeat/module/beat/stats/_meta/data.json#L129) property that we can filter to achieve the same effect.

Also added a functional test for both legacy and metricbeat schemas. The metricbeat archive already existed but is outdated so I updated it with [this script](https://gist.github.com/klacabane/1bcddbee8c04161ac56d2d1ac2a234e2) which aligns the data with the [metricbeat changes](https://github.com/elastic/beats/pull/29577). The metricbeat test also uses the new datastreams.

### Testing
- start one metricbeat process with `elasticsearch-xpack` module enabled
- start one filebeat process that outputs to `console` or `file`
- start one metricbeat process with `beat-xpack` module enabled that monitors the filebeat process

Two cluster with one being `Standalone cluster` should be displayed in the SM Cluster listing view
